test/CppTests/OpenDebug/CrossPlatCpp/LaunchCommand.cs

```diff
@@ -34,12 +37,24 @@ public sealed class CppLaunchCommandArgs : LaunchCommandArgs
         public string MIMode;
 
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public string VisualizerFile;
+        public List<string> VisualizerFile { get; set; }
```

Comment:
You can remove this since if we want to use a visualizer file use one of the other classes instead.

Reply:
yes you will need to fix those based on removing this.